src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/ereg/EregDomene.kt

```diff
@@ -0,0 +1,75 @@
+package no.nav.familie.ef.sak.opplysninger.personopplysninger.ereg
+
```

Comment:
Trenger vi å forholde oss til alt dette? Er det for lagring/fremvisning? 😱

Reply:
Ja jeg tenker vel også att det burde være tilstrekkelig med kun orgnr/navn, men vi kan slette felter senere som vi ikke bruker/skal bruke :)

Reply:
Trenger bare organisasjonsnavnet til visningen. Vurderte å bare ta med orgnummer og navn her, men kom frem til at jeg tok med alt som tjenesten gir i respons.
Kjenner ikke til dataene i ereg så godt, men ut ifra slack-samtaler tror jeg feltet redigertnavn er det som passer best for oss.
Visste heller ikke om redigertnavn kunne være null, så derfor lagde jeg en egen mapper som velger et annet navnefelt dersom redigertnavn ikke finnes.
Tenkte at det var fint å se hva som kommer fra tjenesten og at det mappes over til eget objekt som bare inneholder det som er ønskelig.